api_app/connectors_manager/views.py

```diff
@@ -3,11 +3,30 @@
 
 from rest_framework import generics
 from rest_framework.response import Response
+from django.http.response import Http404
```

Comment:
`from rest_framework.exceptions import NotFound`

and then `raise NotFound()`, dont use `Http404`.

Reply:
same in `analyzers_manager.views`.